responses/fr/HassGetWeather.yaml

```diff
@@ -3,22 +3,200 @@ responses:
-          'lightning-rainy': 'avec de l\'orage et de la pluie',
-          'partlycloudy': 'et un temps partiellement nuageux',
-          'pouring': 'et une pluie battante',
-          'rainy': 'et de la pluie',
-          'snowy': 'et de la neige',
-          'snowy-rainy': 'avec pluie et neige mêlées',
-          'sunny': 'et un temps ensoleillé',
-          'windy': 'avec un temps venteux',
-          'windy-variant': 'et un temps variable et venteux'
-        } %}
-        {{ state.attributes.get('temperature') }} {{ state.attributes.get('temperature_unit') }} {{ weather_condition.get((state.state | string).lower(), "") }}
+        {# Reuse weather condition already formatted and translated #}
+        {# just invert telling the condition then temperature #}
+        {{ state.state }}
+
+        {% set temperature = state.attributes.get('temperature') %}
+        {% if temperature %}
+          {% if temperature < 0 %}
+            {% set temperature_string = 'moins ' ~ (temperature * -1) | string | replace('.', ',') %}
+          {% else %}
+            {% set temperature_string = temperature | string | replace('.', ',') %}
+          {% endif %}
+          {% set temperature_unit = state.attributes.get('temperature_unit', '') %}
+          et il fait {{ temperature_string }} {{ temperature_unit.replace('°C', 'degrés') }}.
+        {% endif %}
+
+      # Starting from here, heavily inspired by the Hungarian template, kudos to RViktor
+      wind: >
+        {% set wind_speed = state.attributes.get('wind_speed') %}
+        {% set wind_speed_unit = state.attributes.get('wind_speed_unit', '') %}
+        {% set wind_bearing = state.attributes.get('wind_bearing') %}
+        {% set wind_gust_speed = state.attributes.get('wind_gust_speed') %}
+
+        {% set wind_scale = '' %} {# Beaufort scale #}
+        {% if wind_speed and wind_speed_unit == 'km/h' %}
+          {% if wind_speed <= 1 %}
+            {% set wind_scale = 'Vent calme' %}
+          {% elif wind_speed <= 5 %}
+            {% set wind_scale = 'Très légère brise' %}
+          {% elif wind_speed <= 11 %}
+            {% set wind_scale = 'Légère brise' %}
+          {% elif wind_speed <= 19 %}
+            {% set wind_scale = 'Petite brise' %}
+          {% elif wind_speed <= 28 %}
+            {% set wind_scale = 'Jolie brise' %}
+          {% elif wind_speed <= 38 %}
+            {% set wind_scale = 'Bonne brise' %}
+          {% elif wind_speed <= 49 %}
+            {% set wind_scale = 'Forte brise' %}
+          {% elif wind_speed <= 61 %}
+            {% set wind_scale = 'Grosse brise' %}
+          {% elif wind_speed <= 74 %}
+            {% set wind_scale = 'Coup de vent' %}
+          {% elif wind_speed <= 88 %}
+            {% set wind_scale = 'Fort coup de vent' %}
+          {% elif wind_speed <= 102 %}
+            {% set wind_scale = 'Tempête' %}
+          {% elif wind_speed <= 117 %}
+            {% set wind_scale = 'Violente tempête' %}
+          {% else %}
```

Comment:
Can you transform this `else` into a `eilf wind_speed > 117`.
I know it should be the same, but I want to avoid showing `vent d'ouragan` if the wind_speed is incorrectly set to `unavailable` or `-1` for example